src/database/backtrack.js

```diff
@@ -0,0 +1,66 @@
+import UserSettings from './models/userSettings';
+import UserArticleLink from './models/userArticleLink';
+import { parseToJson } from '@cofacts/line-bot-log-parser';
+
+async function main() {
+  const parse = async path => {
+    const status = {
+      processedLog: 0,
+      writtenUserSettings: 0,
+      writtenUserArticleLink: 0,
+    };
+
+    await parseToJson(path, async (data, next) => {
+      const timestamp = new Date(data['timestamp']);
+      const userId = data['userId'];
+      const selectedArticleId = data['context.data.selectedArticleId'];
+      const selectedReplyId = data['context.data.selectedReplyId'];
+
+      // console.log(timestamp, userId, selectedArticleId, selectedReplyId);s
+      status.processedLog++;
+      if (userId) {
+        await UserSettings.create({ userId });
+        status.writtenUserSettings++;
+
+        if (selectedArticleId) {
+          if (selectedReplyId) {
```

Comment:
As [we recently discussed](https://g0v.hackmd.io/eIeU2g86Tfu5VnLazNfUvQ#%E8%B3%87%E6%96%99%E8%A1%A8%EF%BC%9AUserArticleLinks), we no longer need to populate `lastRepliedAt` :free:

Reply:
Both branch in this if-else statement does exactly the same thing.

Reply:
Haha I just change the function name, not seeing this